scripts/python/testbedApi.py

```diff
@@ -0,0 +1,329 @@
+# Python testing utility classes and functions
+import paramiko
+import threading
+import sys
+import os
+
```

Comment:
This comment is meant for `scripts/python/testbedApi.pyc` and `scripts/python/objmodel.pyc` files. Looks like they sneaked in despite the `.gitignore` change.